package/utils/audit/Makefile

```diff
@@ -0,0 +1,184 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=audit-userspace
```

Comment:
I would prefer the name `audit` like the folder name. Are there any problems with that?

Reply:
ah ok, this is named `audit-userspace` upstream, then it is fine.

Reply:
I'll do a folder rename on next update, if you want them consistent.